client/src/components/overview/ZoomedImageController.js

```diff
@@ -0,0 +1,191 @@
+import ZoomedImageModel from './ZoomedImageModel.js';
+import './zoomedImage.css';
+
+class ZoomedImageController {
+  constructor(containerId, imageId, resultId = '', zoom = 2.5, overlay = true) {
+    this.zoom = zoom;
+    this.overlay = overlay;
+
+    this.containerId = containerId;
+    this.imageId = imageId;
+    this.resultId = resultId;
+    this._generateResult = (this.resultId === '');
+
+    this.containerClassName = 'img-zoom-container';
+    this.resultClassName = 'img-zoom-result';
+    this.lensClassname = 'img-zoom-lens';
+  }
```

Comment:
I think we can use className as a string. Are you create variable ClassName because you have a special purpose ?

Reply:
Yes, these were originally written in the HTML elements, but rather than manipulating CSS style definitions in Javascript to add/remove style state when changing from zoomed to expanded and back, it seemed cleaner & simpler to just add/remove classes that trigger the application of certain CSS styles.

I still manipulated some CSS style attributes, but it was more limited just to properties that were to be programmatically determined at run time, such as properties based on the rendered size of another HTML element.